src/Content/JsonContent.cs

```diff
@@ -0,0 +1,59 @@
+﻿// <copyright file="JsonContent.cs" company="ProjectBlackmagic">
+// RestfulClient
+// Copyright (c) ProjectBlackmagic. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// Github: https://github.com/ProjectBlackmagic/RestfulClient.
+// </copyright>
+
+using System;
+using System.Net.Http;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace ProjectBlackmagic.RestfulClient.Content
+{
+    /// <summary>
+    /// JsonContent defines a subtype of <see cref="StringContent"/> for JSON content.
+    /// </summary>
+    /// <typeparam name="T">Type of object being serialized</typeparam>
+    public class JsonContent<T> : StringContent
+    {
+        private static readonly Encoding StringEncoding = Encoding.UTF8;
+        private static readonly string MediaType = "application/json";
+
+        private static readonly JsonSerializerSettings DefaultSerializerSettings = new JsonSerializerSettings()
```

Comment:
It is only used in one of the constructors.

Reply:
Yep. And? :)